build.gradle

```diff
@@ -47,8 +48,14 @@ group = 'com.android.volley'
 version = '1.2.0-SNAPSHOT'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion = '25.0.2'
+    compileSdkVersion 28
+    buildToolsVersion = '28.0.3'
+
+    defaultConfig {
+        // Keep in sync with src/main/AndroidManifest.xml
+        minSdkVersion 8
+        targetSdkVersion 28
```

Comment:
sorry for not noticing this before - since this was unset in previous builds, what was the effective value - 1 as well? If so, I'm hesitant to change this to 28 in case that leads to compatibility issues in apps which are using Volley but currently have older targetSdkVersions.

Reply:
It's unclear what the effective previous target was.  But, it's fine for libraries to have a higher target than the app.  The effective target is determined by the application's target, not any libraries.

Reply:
I think you can just omit the targetSdkVersion altogether (at least I was able to patch in your CL and build without it). The minSdkVersion is needed to build the test manifest since it fails to infer it from the AndroidManifest.xml, but the targetSdkVersion can remain unset, which seems like it would preferable just in case it has unexpected impact.